tests/phpunit/integration/includes/TestCase.php

```diff
@@ -45,8 +45,6 @@ public function tear_down() {
 			$_SERVER['HTTPS']
 		);
 
-		$this->clean_up_global_scope();
```

Comment:
Why remove this?

Reply:
`\WP_UnitTestCase_Base::set_up()` already does this for us, so this is unnecessary here